internal/proxy/utils.go

```diff
@@ -51,9 +51,15 @@ func (p *Proxy) processTemplate(w http.ResponseWriter, ns string, requestLogEntr
 	if err != nil {
 		return
 	}
-	data := struct{ Retry int }{Retry: 15}
+	// Jenkins takes around 4 mins to start so start with
+	// 3 min, 1.5 min, ...  until 15 sec
+	// see index.html that implements the exponential backoff retry
```

Comment:
4 minutes is not always the case. I have seen unidling in 2+ minutes. Can we do something like 2 min, 1/2 min, 15 sec, for rest tries 10 sec ?

Reply:
I will reduce the intial delay to 2mins but will keep the `min` 15sec as is. so that we give atleast 15sec for jenkins to become operational.